src/scripts/modules/media/body/body.coffee

```diff
@@ -460,10 +461,14 @@ define (require) ->
     toggleSolution: (e) ->
       $solution = $(e.currentTarget).closest('.solution, [data-type="solution"]')
       $solution.toggleClass('ui-solution-visible')
+      $uiBody = $solution[0].getElementsByClassName('ui-body')[0]
+      $uiBodyLive = $solution[0].getElementsByClassName('ui-body-live')[0]
       if $solution.hasClass('ui-solution-visible')
         $solution.attr('aria-expanded',true)
         $solution.attr('aria-label',"hide solution")
+        $uiBodyLive.innerHTML = $uiBody.innerHTML
```

Comment:
From https://stackoverflow.com/q/10349987 : Rather than creating a new element and moving the HTML, could adding `role=alert` to the element and then unhiding it have the same effect?

From:

> you can add on the fly or have it already on the hidden content, the change from display:none to display:block or the addition of content to the element with role=alert will cause an accessibility event to fire and the content to be announced. an example: [link] dl.dropbox.com/u/377471/tests/ajax.html – Steve Faulkner

Reply:
From: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

>Because of its intrusive nature, the alert role must be used sparingly and only in situations >where the user's immediate attention is required. Dynamic changes that are less urgent should >use a less aggressive method, such as aria-live="polite" or other live region roles.

So I think that `aria-live="polite/assertive"` and setting `display: none;` instead of moving elements would be ok.